src/app/donation-start/donation-start.component.ts

```diff
@@ -954,21 +954,19 @@ export class DonationStartComponent implements AfterContentChecked, AfterContent
     }
   }
 
-  private loadAuthedPersonInfo(id: string, jwt: string) {
-    this.identityService.get(id, jwt).subscribe((person: Person) => {
-      this.personId = person.id; // Should mean donations are attached to the Stripe Customer.
-      this.personIsLoginReady = true;
-
-      if (environment.creditDonationsEnabled && person.cash_balance && person.cash_balance[this.campaign.currencyCode.toLowerCase()]! > 0) {
-        this.creditPenceToUse = parseInt(
-          person.cash_balance[this.campaign.currencyCode.toLowerCase()]!.toString() as string,
-          10
-        );
-        this.stripePaymentMethodReady = true;
-        this.setConditionalValidators();
-      }
+  loadAuthedPersonInfo(id: string, jwt: string) {
+    if(this.identityService) {
+      this.identityService.get(id, jwt).subscribe((person: Person) => {
+        this.personId = person.id; // Should mean donations are attached to the Stripe Customer.
+        this.personIsLoginReady = true;
+        this.prefillRarelyChangingFormValuesFromPerson(person);
+        this.loadFirstSavedStripeCardIfAny(id, jwt);
```

Comment:
I think this will conflict with #1031 . We'll find out for sure when whichever one is merged first and can deal with it then. Good example of why it's so good to make small quick PRs, since if we merge them quickly we get to find out about the conflicts quickly.